### PR TITLE
T15239

### PR DESCRIPTION
**Pull request by katelk28**
Part of design QA for the FNDE apps. Specifically:
- Library-List preset
- Escola preset

There is another [PR](https://github.com/endlessm/endless-content-factory/pull/579) for the same Phabricator ticket in endless-content-factory for the Portal do Professor article page SCSS/CSS.

https://phabricator.endlessm.com/T15239

### CODE REVIEW

File: data/css/escola.scss [hunk 1]
```diff
@@ -203,3 +254,52 @@ where it needs to be */
...
+    .landing {
+      .background {
+        min-height: 250px;
+      }
+
+      .BannerDynamic {
+        &__logo {
+          min-height: 120px;
+          min-width: 400px;
+          font-size: 60px;
```
Comment (tchx84) [inline data/css/escola.scss]: `font-size` doesn't really affect the size of the DynamicLogo,the size you see is only the result of the min-height and min-width you use there.

File: data/css/escola.scss [hunk 2]
```diff
@@ -191,10 +237,15 @@ where it needs to be */
...
-      color: #b3b3b3;
+      color: $card-context-color;
     }
     &__thumbnail {
-      background-color: $base-color;
+      background-color: $background-dark-color;
+    }
+    &.width-d.height-b &__contentFrame,
+    &.width-e.height-b &__contentFrame,
+    &.width-e.height-c &__contentFrame {
```
Comment (tchx84) [inline data/css/escola.scss]: Watch out with the `&` here! `&` means "everything that comes higher in the selector hierarchy", so what you get from:

```
 &.width-e.height-c &__contentFrame {
``` 

is actually:

```
.ArrangementGrid .CardThumb.width-e.height-c .ArrangementGrid .CardThumb__contentFrame {
      padding: 20px;
}
```

Notice the `.ArrangementGrid` in the middle, therefore this selector won't work.

File: data/css/escola.scss [hunk 3]
```diff
@@ -122,16 +132,38 @@ $display-font: 'Raleway Black' !default;
...
+    font-size: 1.25em;
+    color: white;
+    padding: 0em 1.25em 0em 1.25em;
+    font-style: normal;
+  }
+  &__subtitle {
+    margin-top: 1.0em;
+    padding: 0;
+  }
+}
```
Comment (tchx84) [inline data/css/escola.scss]: I think there is something off in composite mode with the no results page:

virtualbox_endlessos_23_02_2017_16_21_56

File: data/css/escola.scss [hunk 4]
```diff
@@ -203,3 +254,52 @@ where it needs to be */
...
+  .set-page,
+  .search-page {
+    .ContentGroupContentGroup,
+    .BannerSearch {
+      &__title {
+        font-size: 30px;
+      }
+    }
+  }
+}
```
Comment (tchx84) [inline data/css/escola.scss]: Just asking here, but was there any plans to improve the look of the search box popup?

virtualbox_endlessos_23_02_2017_16_19_42

File: data/css/escola.scss (hunk 3 above)
Comment (katelk28) [inline data/css/escola.scss]: You're right -- am fixing the visuals a bit.

Comment (xjuan) [inline data/css/escola.scss]: virtualbox_endless dev_28_02_2017_14_18_40

Looks good now!

**Comment by xjuan** [comment]
CR PAss

**Pull request merged**